tests/fixtures/vcr_cassettes/api_unreachable.yml

```diff
@@ -1,7 +1,7 @@
 http_interactions:
 - request:
     method: get
-    uri: http://www.sherpa.ac.uk/romeo/api29.php?jtitle=Journal&qtype=contains
+    uri: http://www.sherpa.ac.uk/romeo/api29.php?jtitle=Journal&qtype=contains&ak=<<<api_key>>>
```

Comment:
Let's try to set the min `vcr` version to 0.5.0 and see what happens with Appveyor.

Reply:
Ok I'll try that